.gitignore

```diff
@@ -38,11 +38,17 @@ __pycache__
 # image files
 *.png
 
+/docs/site/*
+
 # Documentation build directory
 docs/api/
 docs/doxyoutput/
 _build/
 
+# Include image and pdf files inside docs
+!/docs/docs/flowchart-v2.pdf
+!/docs/docs/tests/attach/*.png
```

Comment:
Should these have `/` at the beginning of the path?

Reply:
Same here.